ai_genomics/getters/patents.py

```diff
@@ -1,5 +1,4 @@
 from ai_genomics import bucket_name as BUCKET_NAME
-import pandas as pd
```

Comment:
This was used in the type hint below so now `from ai_genomics.getters.patents import get_ai_genomics_patents` throws a `NameError`